pkg/disk/nodeserver.go

```
@@ -187,10 +196,15 @@ func NewNodeServer(m metadata.MetadataProvider) csi.NodeServer {
 	}
 
 	kataBMIOType := BDF
-	if bmType := os.Getenv("KATA_BM_IO_TYPE"); bmType == DFBus.BusName() {
+	value, err := m.Get(metadata.VmocType)
+	if err != nil {
+		klog.Errorf("get vmoc failed: %+v", err)
+	}
+	if err == nil && value == "true" {
```

else if value == "true". And we should not ignore all errors?